src/components/Card/Card.js

```diff
@@ -1,13 +1,59 @@
-import React from 'react'
+import React, { Component } from 'react'
 
-export default function Card({ cardData }) {
-  return (
-    <div className='card'>
-      <button>Favorite</button>
-      <h1>{cardData.name}</h1>
-      <h1>{cardData.species}</h1>
-      <h1>{cardData.homeworld}</h1>
-      <h1>{cardData.population}</h1>
-    </div>
-  )
+export default class Card extends Component {
+  constructor() {
+    super()
+    this.state = {
+      flipped: false,
+      unflipped: false
+    }
+  }
+
+  flipCard = () => {
+    if (this.state.flipped === true) {
+      this.setState({
+        flipped: false,
+        unflipped: true
+      })
+    } else {
+      this.setState({
+        flipped: true,
+        unflipped: false
+      })
+    }
```

Comment:
This could be simplified to (
let pastFlipped = this.state.flipped
let unflipped = this.state.flipped
let flipped = this.state.unflipped
this.setState({
        flipped: flipped,
        unflipped: unflipped
)